draft-ietf-tls-tls13.md

```diff
@@ -5636,6 +5759,72 @@ application protocols separately ensuring that confidential
 information is not inadvertently leaked.
 
 
+## Replay Attacks on 0-RTT {#replay-0rtt}
+
+Replayable 0-RTT data presents a number of security threats to
+TLS-using applications. Specifically, if applications are not
+engineered to be replay safe (minimally, this means idempotent, but in
+some cases may there might be even stronger conditions), then duplication of
```

Comment:
Past discussions seem to indicate that there are stronger conditions in most cases, not just some.
I think we should consider adding an enumeration of potential other issues, like side channel leakage, DoS/rate-limiting, reordering, etc.  Which actually looks a lot like the next few paragraphs, and...